transferui/src/main/res/layout/item_destination.xml

```diff
@@ -0,0 +1,80 @@
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:paddingStart="@dimen/grid_padding_left"
+    android:paddingTop="@dimen/grid_padding_top"
+    android:paddingEnd="@dimen/grid_padding_right"
+    tools:layout_height="wrap_content"
+    tools:layout_width="match_parent">
+
+    <TextView
+        android:id="@+id/icon"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/grid_margin_left"
+        android:layout_marginTop="@dimen/grid_margin_top"
+        android:background="@drawable/circle"
+        android:fontFamily="@font/icomoon"
+        android:gravity="center"
+        android:textColor="@color/colorPrimary"
+        android:textSize="@dimen/font_default_icon_size"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="\uE000"/>
+
+    <TextView
+        android:id="@+id/title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/grid_margin_left"
+        android:layout_marginTop="@dimen/grid_margin_top"
+        android:textAppearance="@style/TextAppearance.Hyperwallet.Subtitle1"
+        app:layout_constraintLeft_toRightOf="@+id/icon"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="Bank Account"/>
+
+    <TextView
+        android:id="@+id/description_1"
+        android:layout_width="@dimen/description_max_layout_width"
+        android:layout_height="wrap_content"
+        android:textAppearance="@style/TextAppearance.Hyperwallet.Body2"
+        app:layout_constraintLeft_toLeftOf="@id/title"
+        app:layout_constraintTop_toBottomOf="@id/title"
+        tools:text="United States"/>
+
+    <TextView
+        android:id="@+id/description_2"
+        android:layout_width="@dimen/description_max_layout_width"
+        android:layout_height="wrap_content"
+        android:textAppearance="@style/TextAppearance.Hyperwallet.Body2"
+        app:layout_constraintLeft_toLeftOf="@id/title"
+        app:layout_constraintTop_toBottomOf="@id/description_1"
+        tools:text="Editing on 3443"/>
+
+    <ImageView
+        android:id="@+id/item_selected_image"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:contentDescription="@string/destination_item_description"
+        android:paddingStart="@dimen/item_vertical_padding"
+        android:paddingEnd="@dimen/item_vertical_padding"
+        android:paddingBottom="@dimen/grid_padding_bottom"
+        android:src="@drawable/ic_check_14dp"
+        android:visibility="gone"
+        app:layout_constraintBottom_toTopOf="@+id/divider"
```

Comment:
Please change from `@+id/divider` to `@+id/title` so that check mark will align to the title not in edge of the UI

Reply:
address here https://github.com/hyperwallet/hyperwallet-android-ui-sdk/pull/84